src/components/MapEmbed.js

```diff
@@ -34,28 +44,29 @@ const MapEmbed = (props) => {
 		location = '660 4th Street #119 San Francisco CA 94107, USA';
 	}
 
-	if (!location) {
+	if (mapPlatform === 'google') {
+		return (
+			<GoogleMap
+				location={location}
+				className={className}
+				zoom={zoom}
+				type={type}
+				height={height}
+			/>
+		);
+	} else if (!location) {
 		return <></>;
 	}
 
-	const style = { border: 0, height, width: '100%' };
-	const baseUrl = 'https://maps.google.com/maps';
-
-	const params = new URLSearchParams({
-		q: location,
-		z: zoom || 10,
-		t: type || 'm',
-		output: 'embed',
-	});
-
-	const srcURL = baseUrl + '?' + params.toString();
 	return (
-		<iframe
-			src={srcURL}
-			style={style}
+		<LeafletMap
+			location={location}
+			latitude={latitude}
+			longitude={longitude}
 			className={className}
-			title={location}
-		></iframe>
+			zoom={zoom}
+			height={height}
+		/>
```

Comment:
I think this map selection can be a little cleaner. Make sure a `mapPlatform` is selected and be explicit. The setting itself will have OSM as a default, so you don't have to worry about it being empty (or shouldn't). If we ever change defaults, we want to make sure we don't have code that assumes too much at this point. Also, if no `location` fail quick in an `if` statement that precedes this `mapPlatform` conditional.

Reply:
I believe has been completed. I have removed the assertion for mapPlatform and ensured we are not assuming the default at this level. Also moved location fail earlier in the if statement